Shared/Enums.cs

```diff
@@ -529,7 +529,7 @@ public enum Monster : ushort
     WereTiger = 336, // Done (DG) - USE AI 113 (BloodBaboon) for Attack2. 
     KingHydrax = 337, //TODO - BOSS AI
     Hydrax = 338, // Done (DG) - No AI
-    HornedMage = 339,
+    HornedMage = 339,// Done (DG)
```

Comment:
Please remove "(DG)" from these comments. This is indicating the person who added the AI